infra/src/api/io_services_cms/v1/policy.xml

```diff
@@ -20,6 +20,12 @@
         <set-header name="x-user-email" exists-action="override">
             <value>@(context.User.Email)</value>
         </set-header>
+        <set-header name="x-user-groups-selc" exists-action="override">
```

Comment:
I propose

```xml
<choose>
    <when condition="@(context.Subscription.Id.StartsWith("MANAGE-GROUP-")">
        <set-header name="x-user-groups-selc" exists-action="override">
            <value>@(context.Subscription.Id.Substring("MANAGE-GROUP-".Length))</value>
        </set-header>
   </when>
</choose>
```

Reply:
In this case, if the client sets this header (externally) and the condition is not met, the header valued by the client would be propagated to the backend, which we want to avoid since this header is meant to be handled only “internally”